### ISSUE TITLE
Fully implement learning from pixels

### ISSUE THREAD

**Issue by sholtodouglas** [sholtodouglas/learning_from_play#16]
- [x]  Optimize DL for images 
- [x]  Find largest viable batch_size
- [x] Include pixels pathway through the train loop
- [x]  Implement spatial softmax https://arxiv.org/abs/1504.00702

**Comment by sholtodouglas** [comment]
Just need to stabilize gradients
